moto/panorama/models.py

```diff
@@ -0,0 +1,304 @@
+        name: str,
+        network_configuration: Optional[Dict[str, Any]],
+        tags: Optional[Dict[str, str]],
+    ) -> None:
+        self.__device_aggregated_status_manager = ManagedState(
+            model_name=f"panorama::device_{name}_aggregated_status",
+            transitions=[("AWAITING_PROVISIONING", "PENDING"), ("PENDING", "ONLINE")],
+        )
+        self.__device_provisioning_status_manager = ManagedState(
+            model_name=f"panorama::device_{name}_provisioning_status",
+            transitions=[
+                ("AWAITING_PROVISIONING", "PENDING"),
+                ("PENDING", "SUCCEEDED"),
+            ],
+        )
+        self.account_id = account_id
+        self.region_name = region_name
+        self.description = description
+        self.name = name
+        self.network_configuration = network_configuration
+        self.tags = tags
+
+        self.arn = (
+            f"arn:aws:panorama:{self.region_name}:{self.account_id}:device/{self.name}"
+        )
+        self.device_id = f"device-{hash_device_name(name)}"
+        self.iot_thing_name = ""
+
+        self.alternate_softwares = [
+            {"Version": "0.2.1"},
+        ]
+        self.brand: Literal["AWS_PANORAMA", "LENOVO"] = "AWS_PANORAMA"
+        self.created_time = datetime.now(tzutc())
+        self.last_updated_time = datetime.now(tzutc())
+        self.current_networking_status = {
+            "Ethernet0Status": {
+                "ConnectionStatus": "CONNECTED",
+                "HwAddress": "8C:0F:5F:60:F5:C4",
+                "IpAddress": "192.168.1.300/24",
+            },
+            "Ethernet1Status": {
+                "ConnectionStatus": "NOT_CONNECTED",
+                "HwAddress": "8C:0F:6F:60:F4:F1",
+                "IpAddress": "--",
+            },
+            "LastUpdatedTime": datetime.now(tzutc()),
+            "NtpStatus": {
+                "ConnectionStatus": "CONNECTED",
+                "IpAddress": "91.224.149.41:123",
+                "NtpServerName": "0.pool.ntp.org",
+            },
+        }
+        self.current_software = "6.2.1"
+        self.device_connection_status: Literal[
+            "ONLINE", "OFFLINE", "AWAITING_CREDENTIALS", "NOT_AVAILABLE", "ERROR"
+        ] = "ONLINE"
+        self.latest_device_job = {"JobType": "REBOOT", "Status": "COMPLETED"}
+        self.latest_software = "6.2.1"
+        self.lease_expiration_time = datetime.now(tzutc()) + timedelta(days=5)
+        self.serial_number = "GAD81E29013274749"
+        self.type: Literal[
+            "PANORAMA_APPLIANCE_DEVELOPER_KIT", "PANORAMA_APPLIANCE"
+        ] = "PANORAMA_APPLIANCE"
+
+    @property
+    def device_aggregated_status(self) -> DEVICE_AGGREGATED_STATUS_TYPE:
+        _device_aggregated_status: DEVICE_AGGREGATED_STATUS_TYPE = self.__device_aggregated_status_manager.status  # type: ignore[assignment]
+        self.__device_aggregated_status_manager.advance()
+        return _device_aggregated_status
```

Comment:
```suggestion
        self.__device_aggregated_status_manager.advance()
        return self.__device_aggregated_status_manager.status
```
Looks like we can simplify this a bit, without the intermediary variable. Same with the provisioning_status below

Reply:
Done: https://github.com/getmoto/moto/pull/6948/commits/fad936cc1e06b8f5c3c868db53ef0fc63a8180e3